### PR TITLE
Contribution guidelines

### PR DESCRIPTION
**Pull request by thomas-lamiaux**
A presentation of the project and basic contribution guidelines. 
They are not perfect nor complete, but I would like to merge something relatively quickly, 
so that people can start looking at it and contributing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,113 @@
...
+- How-to should provide users practical answers to practical problems that they can refer to when working
+- By nature, the documentation should be mostly horizontal, which should:
+  - make it easy to navigate and to find specific information
+  - prevent users from having to read a bunch documentation to be able to read a specific tutorial
+  - make it possible to build gradually, making new tutorials and how-to available as we progress
+  - allow differentiated learning: depending on your background or objective you can navigate the
+    documentation differently, potentially reading different tutorials.
+- It will enable us to showcase all that is possible in Coq's ecosystem
+- It should be easy to maintain as once fully written a tutorials or how-to should have any reason to change,
+  except if the associated feature changes
```
Comment (Zimmi48) [inline README.md:66]:
```suggestion
  except if the associated feature or known best practices change.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,113 @@
...
+
+### Plan
+This project will necessarily have to be a collaborative undertaking considering how much work there is to do,
+and the richness and diversity of our ecosystem.
+Yet, as the tutorials can mostly be designed independently, by combining the different expertise of the different communities,
+there is good hope to quickly get to a good documentation, and we welcome contributions.
+As a base for work, we have established an [informal list](https://github.com/Zimmi48/platform-docs/blob/main/draft_structure_doc.md)
+of tutorials and how-to it could be interesting to have.
```
Comment (Zimmi48) [inline README.md:74]:
```suggestion
of tutorials and how-tos it could be interesting to have.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,113 @@
...
+but it should already give an idea of the potential of this project.
+
+
+
+
+## How to contribute to the documentation
+
+There is a lot of work to be done before having a comprehensive documentation for Coq and its Platform, and we welcome contributions.
+
+There are different possible way to contribute depending on your time and technical skills:
```
Comment (Zimmi48) [inline README.md:85]:
```suggestion
There are different possible ways to contribute depending on your time and technical skills:
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,113 @@
...
+- As users do not hesitate to gives us feedbacks on the project on the dedicated <span style="color:red"><u>zulip chan</u></span>
```
Comment (Zimmi48) [inline README.md:86]:
```suggestion
- As user, do not hesitate to gives us feedback on the project on the dedicated <span style="color:red"><u>Zulip stream</u></span>
```

Should we create the stream now?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,113 @@
...
+
+### Writing Tutorials and How-to
+If you would like to write a tutorial 
+If you have an idea for a tutorial or how-to, you can create a discussion on the dedicated <span style="color:red"><u>zulip chan</u></span>
+to get feedback on your idea, through the writing and to reach others that may be interested people.
+If there is one, you can also create a discussions on a more specific zulip chan, for instance, if you are working on tutorials for the package
+Equation, you can create a discussions on the associated chan.
+
+Once you have a plan and some content, you can create a draft pull request to make your code accessible
+and get feedbacks on it while you (and others) progress on it.
```
Comment (Zimmi48) [inline README.md:103]:
```suggestion
and get feedback on it while you (and others) progress on it.
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,113 @@
...
+
+> [!WARNING]
+> Before starting to work on a tutorial or a how-to and invest time into it, check if it is not already existing,
+> or if someone hasn't already started working on it, either by creating a discussion on the zulip chan or a draft pull request about it.
```
Comment (Zimmi48) [inline README.md:107]:
```suggestion
> or if someone hasn't already started working on it, either by creating a discussion on the Zulip stream or a draft pull request about it.
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,113 @@
...
+
+> [!WARNING]
+> A lot of stuffs have already be written about Coq, it can make sense to reuse some of the content. 
+> If you wish to do so, be careful that you are indeed allowed by the copyrights.
```
Comment (Zimmi48) [inline README.md:111]:
```suggestion
> If you wish to do so, be careful that you are indeed allowed by the copyright owners or the content licenses.
```

**Comment by thomas-lamiaux** [comment]
> Shouldn't this last part be in the contributing guide?
You were right. I partially splitted it. I'll merge now.

**Pull request merged**